### DISCUSSION TITLE
Problem faced with 2FA and billing info

### DISCUSSION THREAD

**Discussion by A-nemon-e** [community/community#83058] [GitHub Education] [answered]
### Hi everyone,

Hi, I keep receiving requests to enable 2FA and complete my billing information although I have already get these done and have signed out and in for several times. Even cleared cookies.
I chose the authenticator application for 2FA but didn't setup other options (like SMS, for my country not listed). Could this be the reason? Or should I just wait for some time to re-apply?
Moreover, my school only offer an electronic proof. No date on student ID card, so the only way is to upload an image of that proof.
Below is all I receive:

> The item you uploaded is insufficient to demonstrate your current academic status.
> Please see suggestions.
> Suggested proof to expedite your approval:
> 
>     -You must secure your GitHub account by enabling 2FA and complete your billing information.
> 
>     -Please secure your GitHub account with two-factor authentication. You may need to log out and log back in to GitHub before re-applying.
> 
>     -You are unlikely to be verified until you have completed your GitHub billing information with your full name exactly as it appears in your academic affiliation document. You do not have to add a payment method. You may need to log out and log back in to GitHub before re-applying.
> 
>     -Please consider using your device camera to submit academic affiliation documents. Uploaded images are more easily manipulated and are therefore less trustworthy.

**Comment by AhmadFida2** [comment]
I also have the same error kindly help please.

**Comment by meowthrower** [comment]
I'm also having the same issue.

**Reply by meowthrower** [reply, marked as answer]
I tried it again a few hours later and it worked

**Reply by A-nemon-e** [reply]
That's it. Just hang on for half a day and re-apply.

**Comment by arijit-topcoder** [comment]
Same with me, not processing the application

**Comment by VJ-13** [comment]
Same issue
